mne/decoding/receptive_field.py

```diff
@@ -0,0 +1,197 @@
+import numpy as np
+from .base import get_coefs, BaseEstimator, _check_estimator
+from ..utils import warn
+
+
+class ReceptiveField(BaseEstimator):
+    """Fit a receptive field model.
```

Comment:
... or at least mention STRF in a second line. Help bring in the neural folks :)